examples/with-storybook-typescript/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "with-storybook",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0"
+  },
+  "license": "ISC",
+  "devDependencies": {
+    "@storybook/addon-actions": "5.3.19",
+    "@storybook/addon-links": "5.3.19",
+    "@storybook/addons": "5.3.19",
+    "@storybook/preset-typescript": "3.0.0",
+    "@storybook/react": "5.3.19",
+    "@types/node": "14.0.13",
+    "@types/react": "16.9.38",
+    "babel-loader": "^8.0.5",
+    "fork-ts-checker-webpack-plugin": "5.0.4",
+    "ts-loader": "7.0.5",
```

Comment:
Why are these 2 packages added?

Reply:
It was required by @storybook/preset-typescript (https://github.com/storybookjs/presets/tree/master/packages/preset-typescript#installation)